src/language.json

```diff
@@ -7,6 +7,7 @@
         "enter_old_password": "Enter Old Password",
         "confirm_password": "Please confirm your password",
         "update_password": "Update Password",
+        "update_username": "Update Username",
```

Comment:
Can you move this line to `scripts/generate-lang.js` to the correct category. We don't want to break any translations made by users

Reply:
Yeah, no problem. I switched those out.